addons/mail/static/src/discuss/call/common/call_action_list.xml

```diff
@@ -88,6 +88,15 @@
                     </div>
                 </button>
             </div>
+            <div t-if="isSmall and store.settings.use_push_to_talk and isOfActiveCall" class="d-flex align-items-center flex-wrap justify-content-between p-2">
+                <button class="o-discuss-CallActionList-pushToTalk btn btn-primary d-flex w-100 border-0 shadow-none"
+                    aria-label="Push to talk"
+                    title="Push to talk"
```

Comment:
No title when it's the same as the text visually present and it doesn't truncate like here.
`title` adds nothing but annoyance on mouse over, so drop it.